Agents Graphql endpoints fail with NoSuchMethodError

I followed the instructions in the README and successfully started an agent (e.g. `./gradlew :news-agent:bootRun`). Then I opened the GraphQL endpoint (`http://localhost:8080/graphiql?path=/graphql`) and sent the GraphQL query as specified in the README. The GraphQL request ended up with an NoSuchMethodError. I repeated the whole thing with the weather agent. Same result:

```
java.util.concurrent.CompletionException: java.lang.NoSuchMethodError: 'java.lang.Object com.expediagroup.graphql.generator.execution.FlowSubscriptionExecutionStrategy.fetchField(graphql.execution.ExecutionContext, graphql.execution.ExecutionStrategyParameters)'
        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[na:na]
        at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1194) ~[na:na]
        at java.base/java.util.concurrent.CompletableFuture.thenCompose(CompletableFuture.java:2341) ~[na:na]
        at graphql.GraphQL.parseValidateAndExecute(GraphQL.java:482) ~[graphql-java-21.5.jar:na]
        at graphql.GraphQL.lambda$executeAsync$9(GraphQL.java:440) ~[graphql-java-21.5.jar:na]
        at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1187) ~[na:na]
        at java.base/java.util.concurrent.CompletableFuture.thenCompose(CompletableFuture.java:2341) ~[na:na]
        at graphql.GraphQL.executeAsync(GraphQL.java:428) ~[graphql-java-21.5.jar:na]
        at graphql.GraphQL.execute(GraphQL.java:366) ~[graphql-java-21.5.jar:na]
        at com.expediagroup.graphql.server.execution.GraphQLRequestHandler.executeSubscription(GraphQLRequestHandler.kt:159) ~[graphql-kotlin-server-8.2.1.jar:8.2.1]
        at com.expediagroup.graphql.server.execution.subscription.GraphQLWebSocketServer$handleSubscription$2$3$1$subscriptionJob$1.invokeSuspend(GraphQLWebSocketServer.kt:138) ~[graphql-kotlin-server-8.2.1.jar:8.2.1]
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[kotlin-stdlib-2.0.20.jar:2.0.20-release-360]
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104) ~[kotlinx-coroutines-core-jvm-1.8.1.jar:na]
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584) ~[kotlinx-coroutines-core-jvm-1.8.1.jar:na]
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:811) ~[kotlinx-coroutines-core-jvm-1.8.1.jar:na]
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:715) ~[kotlinx-coroutines-core-jvm-1.8.1.jar:na]
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:702) ~[kotlinx-coroutines-core-jvm-1.8.1.jar:na]
Caused by: java.lang.NoSuchMethodError: 'java.lang.Object com.expediagroup.graphql.generator.execution.FlowSubscriptionExecutionStrategy.fetchField(graphql.execution.ExecutionContext, graphql.execution.ExecutionStrategyParameters)'
        at com.expediagroup.graphql.generator.execution.FlowSubscriptionExecutionStrategy.createSourceEventStream(FlowSubscriptionExecutionStrategy.kt:114) ~[graphql-kotlin-schema-generator-8.2.1.jar:8.2.1]
        at com.expediagroup.graphql.generator.execution.FlowSubscriptionExecutionStrategy.execute(FlowSubscriptionExecutionStrategy.kt:68) ~[graphql-kotlin-schema-generator-8.2.1.jar:8.2.1]
        at graphql.execution.Execution.executeOperation(Execution.java:162) ~[graphql-java-21.5.jar:na]
        at graphql.execution.Execution.execute(Execution.java:104) ~[graphql-java-21.5.jar:na]
        at graphql.GraphQL.execute(GraphQL.java:568) ~[graphql-java-21.5.jar:na]
        at graphql.GraphQL.lambda$parseValidateAndExecute$13(GraphQL.java:487) ~[graphql-java-21.5.jar:na]
        at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1187) ~[na:na]
        ... 15 common frames omitted
```


I suspect that there is a glitch with the GraphQL versions. The Gradle file `multimodule-config.gradle.kts` defines dependencies to `org.eclipse.lmos:arc-graphql-spring-boot-starter` and `com.graphql-java:graphql-java:21.5`. But the package `org.eclipse.lmos:arc-graphql-spring-boot-starter` has a dependency on `com.expediagroup:graphql-kotlin-spring-server:8.2.1`, which also brings `com.graphql-java:graphql-java`, but in version `22.1.`

As soon as I remove the dependency `com.graphql-java:graphql-java:21.5` from `multimodule-config.gradle.kt` I can successfully send GraphQL requests.

Am I missing something here? Otherwise I can prepare a PR.

--- a/buildSrc/src/main/kotlin/multimodule-config.gradle.kts
+++ b/buildSrc/src/main/kotlin/multimodule-config.gradle.kts
@@ -96,7 +96,6 @@ dependencies {
     implementation("org.eclipse.lmos:arc-ollama-client:$arcVersion")
     implementation("org.eclipse.lmos:arc-reader-html:$arcVersion")
     implementation("org.eclipse.lmos:arc-graphql-spring-boot-starter:$arcVersion")
-    implementation("com.graphql-java:graphql-java:21.5")
 
     // Kotlin Dependencies
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:$kotlinXVersion")

